modules/oadp-backup-restore-cr-issues.adoc

```diff
@@ -48,10 +48,17 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
 $ oc delete backup <backup> -n openshift-adp
 ----
 +
-You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded  files to object storage.
+You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded files to object storage.
 
 . Create a new `Backup` CR.
 
+. View the Velero backup details
++
+[subs="+quotes"]
```

Comment:
```suggestion
[source,terminal]]
```

Reply:
@GroceryBoyJr, the replaceable syntax wasn't working when added with [source, terminal]. Isn't it recommended here to use [subs="+quotes"]?